src/frontend/components/LTINav/index.spec.tsx

```diff
@@ -0,0 +1,167 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import { wrapInIntlProvider } from '../../utils/tests/intl';
+import React from 'react';
+import { LTINav } from './index';
+import {
+  documentMockFactory,
+  videoMockFactory,
+} from '../../utils/tests/factories';
+import { uploadState, Video } from '../../types/tracks';
+import { Document } from '../../types/file';
+import { Router } from 'react-router-dom';
+import { PLAYER_ROUTE } from '../routes';
+import { DASHBOARD_ROUTE } from '../Dashboard/route';
+import { modelName } from '../../types/models';
+import { createMemoryHistory } from 'history';
+
+let mockModelName: any;
+let mockCanUpdate: boolean;
+let mockMaintenance: boolean;
+jest.mock('../../data/appData', () => ({
+  appData: {
+    get modelName() {
+      return mockModelName;
+    },
+  },
+  getDecodedJwt: () => ({
+    permissions: {
+      can_update: mockCanUpdate,
+    },
+    maintenance: mockMaintenance,
+  }),
+}));
+
+describe('<LTINav />', () => {
+  const history = createMemoryHistory();
+  const testLTINav = (object: Video | Document) =>
+    render(
+      wrapInIntlProvider(
+        <Router history={history}>
+          <LTINav object={object} />
+        </Router>,
+      ),
+    );
+  describe('Video', () => {
+    it('navigates between routes when video is ready', () => {
+      const video = videoMockFactory({
+        upload_state: uploadState.READY,
+      });
+      mockModelName = modelName.VIDEOS;
+      mockCanUpdate = true;
+      mockMaintenance = false;
+
+      testLTINav(video);
+
+      fireEvent.click(screen.getByRole('link', { name: /dashboard/i }));
+      expect(history.location.pathname).toBe(DASHBOARD_ROUTE(modelName.VIDEOS));
+
+      fireEvent.click(screen.getByRole('link', { name: /preview/i }));
+      expect(history.location.pathname).toBe(PLAYER_ROUTE(modelName.VIDEOS));
+    });
+
+    it('hides dashboard and playlist links when user is not permitted to update video', () => {
```

Comment:
```suggestion
it('hides dashboard and preview links when user is not permitted to update video', () => {```
did you mean preview instead of playlist ?

Reply:
oops, sorry, playlist comes from my next PR :sweat: